tests/system/test_universe_domain.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+
+# Note(TODO): Uncomment these test cases once showcase library is updated
```

Comment:
Please fix. We can update the minimum version of `google-api-core` if necessary.

Reply:
addressed!